cmd/crio/main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"path/filepath"
 	"sort"
 	"strings"
+	"syscall"
```

Comment:
ew, dead-end dependency ;).  Can we use `golang.org/x/sys/unix` behind the appropriate build tags?  I dunno what Windows expects for this (if CRI-O even builds for Windows?).

Reply:
I can try that, I assumed it was safe to use "syscall" as we are already using it in few different places  (and I am not sure how it is supposed to work on Windows)